lib/govuk_index/presenters/expanded_links_presenter.rb

```diff
@@ -20,6 +20,10 @@ def people
       slugs("people", "/government/people/")
     end
 
+    def roles
+      slugs("roles", "/government/ministers/")
```

Comment:
For now, only ministerial roles get base paths. In the future we might have to support more types of roles, although that's not necessary at the moment.